src/otx/__init__.py

```diff
@@ -3,5 +3,5 @@
 # Copyright (C) 2021-2023 Intel Corporation
 # SPDX-License-Identifier: Apache-2.0
 
-__version__ = "1.4.4rc1"
+__version__ = "1.4.4"
```

Comment:
Can we upload same versions multiple times on internal PyPI?

Reply:
no, but I think we can delete uploaded files in the internal PyPI. I've asked the way to do that to the internal team but did not get the answer yet.